activities/documentation/move-merge-delete-pages.md

```diff
@@ -0,0 +1,62 @@
+---
+type: resource
+---
+
+# Moving, merging and deleting pages
+
+As an [open organization](../../organization/cultural-values.md) dedicated to the long term sustainability of the public code ecosystem, our ambition is to reliably serve readers with the information they were looking for.
+This means we try never to have a url resolve to a `404:Page not found`.
+
+That being said, our content will evolve over time, and we may need to move, merge and delete pages.
+What follows is how we use redirects to handle these different situations.
+
+## How redirects work in Jekyll
+
+There are types of redirects in Jekyll - a redirect from, and a redirect to.
+
+A `redirect_to` is an instruction on the old page, that when the old url is clicked, the redirect instruction on the old page automatically forwards to the new page.
+This works a bit like post arriving at your old home, and being forwarded to your new address.
+A redirect to can only redirect to one other new page.
+Note: this mean a file needs to be kept in the old location.
+
+A `redirect_from` is an instruction in the new page.
+This works a bit like updating everyone's address book with your new address.
+A `redirect_from` can rewrite urls from many other old pages.
+Note: this only works with files in the same repository (we can't change someone else's address book!) - but means the old file can be deleted.
+
+It is technically possible to simultaneously put a `redirect_to` in the old page and `redirect_from` in the new page, providing both pages are in the same repository, but it does not add any value.
+
+## How to add a redirect to a page
+
+Redirect instructions can be included in the front matter by including one of these alternatives:
+
+`redirect_to: [url link]`
+
+`redirect_from: [url link]`
+
+```
+redirect_from:
+    - [url link 1]
+    - [url link 2]
+    - [url link 3]
+```
+
+## Redirecting pages
+
+When a page url changes, but it remains in the same repository (it is moved to a different directory or is renamed) a `redirect_from` should be used.
+
+When a page is moved to a different repository a `redirect_to` should be used.
+
+When a page is merged with another page, use `redirect_from` if it is in the same repository, and a `redirect_to` if it is in a different repository.
+
+When content outgrows its location and moves to its own page, consider using a hyperlink.
+
+When a page is split into new pages, and the old page becomes redundant, consider whether it could become an index or short stub referencing the new pages.
```

Comment:
There's no need to keep the old page as an index or stub if the old page and the new pages are all already included in an index.
```suggestion
When a page is split into new pages, and the old page becomes redundant, redirect to the index that lists the new pages.
```